packages/components/pages/f-self-exclusion/src/tenants/da-DK.js

```diff
@@ -19,6 +19,7 @@ const messages = {
         text1Show: 'We’ll show alcohol items.',
         text1Temporary: 'We\'ll exclude items containing alcohol from your menus on Menulog for six months.',
         text1Permanent: 'We\'ll permanently exclude items containing alcohol from your menus on Menulog.',
+        text1Expired: 'You have successfully removed the alcohol self-exclusion. It will take 24 hours for your change to take effect.',
```

Comment:
I'm sure this is more a question for your PM, but this messaging doesn't appear to match the purpose? Expiry != removal

Reply:
This is as intended.